Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: build docs test
+.PHONY: build docs test test-xpu
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)